### PR TITLE
Fixes #1101 Field can be converted to local variable

### LINKED ISSUE #1101: Fix warning "Field can be local"
## Description
There are some "Field can be local" warnings in the project.

## Mocks
warningfieldcanbelocal

## Acceptance Criteria
- [ ] Converting specific fields to local variables

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by PiumiHR**
### Description
In 8 java classes there were fields which can be covered as a local variables. This is a warning of Class Structure.

Fixes #1101 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
I Inspected the code. After doing changes it did not show the previous warnings. And I compiled the project and run it on the device after doing changes.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java [hunk 1]
```diff
@@ -46,8 +43,8 @@ protected void onCreate(Bundle savedInstanceState) {
         context = StartActivity.this;
         preferences = PreferenceManager.getDefaultSharedPreferences(getBaseContext());
         hasPreviouslyStarted = preferences.getBoolean(getString(R.string.preferences_has_previously_started), false);
-        hasPreviouslyCustomized = preferences.getBoolean(getString(R.string.preferences_has_previously_customized), false);
-        newUserButton = (Button) findViewById(R.id.newUserButtonFirstPage);
+    //    boolean hasPreviouslyCustomized = preferences.getBoolean(getString(R.string.preferences_has_previously_customized), false);
```
Comment (Rimjhim28) [inline PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java]: Please recheck. You have commented out the line.

File: PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java [hunk 2]
```diff
@@ -77,9 +76,9 @@ public void onCreate(Bundle savedInstanceState) {
         if(new VocabMatchSessionManager(this).isVocabMatchOpened()) {
             startActivity(new Intent(GameActivity.this, VocabMatchGameActivity.class));
         }
-        if (savedInstanceState != null) {
-            isStateChanged = true;
-        }
+       /* if (savedInstanceState != null) {
+            boolean isStateChanged = true;
+        }*/
```
Comment (Rimjhim28) [inline PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: You have comment out the if-else block. Please revert this.

**Comment by PiumiHR** [comment]
@Rimjhim28 Thank you. I removed unused statements.

**Comment by PiumiHR** [comment]
@Rimjhim28 Codacy/PR check shows 2 errors regarding that 2 variables. If android studio says these 2 variables can be used as local variables then we can only use them within that specific function.  In current class those variables are unused. cant we create them when we needed?

**Comment by Rimjhim28** [comment]
As far as I remember these variables were actually used in the code, but with recent changes the use might have been modified. @geekanamika @m-murad can you please cross check once.

**Comment by PiumiHR** [comment]
Thank you @Rimjhim28 I will wait for other responses

**Comment by PiumiHR** [comment]
Yes, that's right. Thank you @Vaibhavi24

**Pull request closed without merging**